scripts/appendComponentList.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I think this would be more portable and maintainable for contributors if this script was written in nodejs, thoughts?

Reply:
ok